prototyper/prototyper/src/platform/mod.rs

```diff
@@ -78,23 +78,57 @@ impl Platform {
     }
 
     pub fn init(&mut self, fdt_address: usize) {
-        self.info_init(fdt_address);
-        self.sbi_init();
-        trap_stack::prepare_for_trap();
-        self.ready.swap(true, Ordering::Release);
-    }
-
-    fn info_init(&mut self, fdt_address: usize) {
         let dtb = parse_device_tree(fdt_address).unwrap_or_else(fail::device_tree_format);
         let dtb = dtb.share();
 
         let root: serde_device_tree::buildin::Node = serde_device_tree::from_raw_mut(&dtb)
             .unwrap_or_else(fail::device_tree_deserialize_root);
         let tree: Tree = root.deserialize();
 
-        // Get console device, init sbi console and logger
+        // Get console device, init sbi console and logger.
         self.sbi_find_and_init_console(&root);
+        // Get clint and reset device, init sbi ipi, reset, hsm and rfence.
+        self.sbi_init_ipi_reset_hsm_rfence(&root);
+        // Initialize pmu extension
+        self.sbi_init_pmu(&root);
+        // Get other info
+        self.sbi_mics_init(&tree);
```

Comment:
```suggestion
        self.sbi_misc_init(&tree);
```
Will it be better if we rename this function into `sbi_misc_init`? 'Misc' can be considered a common abbreviation for 'miscellaneous'.

Reply:
Yes, it was my mistake, I have corrected the spelling.